waiter/src/waiter/core.clj

```diff
@@ -701,6 +701,11 @@
                       _ (password-store/check-empty-passwords passwords)
                       processed-passwords (mapv #(vector :cached %) passwords)]
                   processed-passwords))
+   :profile->defaults (pc/fnk [[:settings profile-config service-description-defaults]]
+                        (let [custom-profiles (pc/for-map [[profile {:keys [service-parameters]}] profile-config]
+                                                  (name profile)
+                                                  (merge service-description-defaults service-parameters))]
+                          (assoc custom-profiles "" service-description-defaults)))
```

Comment:
I'm not in love with empty string as a key.

Reply:
How do you feel about using a keyword, e.g. `:default`?

Reply:
Removed the `:default` entry and renamed to `profile->overrides`.